### PR TITLE
Backport Go 1.19.8 CVE fixes to 1.17

### PR DESCRIPTION
**Pull request by bhavi-koduru**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: projects/golang/go/1.17/patches/0016-go-1.17.13-eks-net-textproto-avoid-overpredic.patch
Comment (bhavi-koduru) [inline projects/golang/go/1.17/patches/0016-go-1.17.13-eks-net-textproto-avoid-overpredic.patch:1]: Merge conflicts at 82 and 101 are resolved by accepting upstream changes.

File: projects/golang/go/1.17/patches/0105-go-1.17.13-eks-add-Cut-method-to-bytes-package.patch
Comment (bhavi-koduru) [inline projects/golang/go/1.17/patches/0105-go-1.17.13-eks-add-Cut-method-to-bytes-package.patch:1]: Source commit: https://github.com/golang/go/commit/8e36ab055162efa6f67f3b9ee62f625ac8874901
This utility patch supports Cut method in the bytes package which is utilized by 0034-go1.16.15-eks-net-textproto-avoid-overpredic.patch
Also, added the pkg to api/go1.16.txt  for the changes to kick in.

File: projects/golang/go/1.17/rpmbuild/SPECS/golang.spec [hunk 1]
```diff
@@ -164,6 +164,12 @@ Patch8:   0008-go-1.17.13-eks-multipart-limit-memory-inode-consumption.patch
 Patch9:   0009-go-1.17.13-eks-path-filepath-do-not-clean-relative-path.patch
 Patch10:  0010-go-1.17.13-eks-update-bundled-golang-org.patch
 Patch11:  0011-go-1.17.13-eks-net-http-update-bundled-golang.patch
+Patch12:  0012-go-1.17.13-eks-go-scanner-reject-large-line-a.patch
+Patch13:  0013-go-1.17.13-mime-multipart-avoid-excessive.patch
+Patch14:  0014-go-1.17.13-eks-net-textproto-mime-multipart-i.patch
+Patch15:  0015-go-1.17.13-eks-mime-multipart-limit-parsed-mi.patch
+Patch105:  0105-go-1.17.13-eks-add-Cut-method-to-bytes-package.patch
```
Comment (bhavi-koduru) [inline projects/golang/go/1.17/rpmbuild/SPECS/golang.spec:171]: Patches are applied in this order. Hence, Utility patch is placed before the CVE fix patch.

File: projects/golang/go/1.17/patches/0015-go-1.17.13-eks-mime-multipart-limit-parsed-mi.patch
[the remark this answers is not in the document]
Comment (zafs23) [reply projects/golang/go/1.17/patches/0015-go-1.17.13-eks-mime-multipart-limit-parsed-mi.patch:1]: @bhavi-koduru this should be go 1.17*

File: projects/golang/go/1.17/patches/0105-go-1.17.13-eks-add-Cut-method-to-bytes-package.patch
Comment (zafs23) [reply projects/golang/go/1.17/patches/0105-go-1.17.13-eks-add-Cut-method-to-bytes-package.patch:1]: @bhavi-koduru the description should match for go1.17

File: projects/golang/go/1.17/patches/0012-go-1.17.13-eks-go-scanner-reject-large-line-a.patch
Comment (bhavi-koduru) [inline projects/golang/go/1.17/patches/0012-go-1.17.13-eks-go-scanner-reject-large-line-a.patch:1]: Merge conflicts are resolved by accepting only the tests introduced in this fix.

**Pull request merged**